charts/aws-efs-csi-driver/values.yaml

```diff
@@ -34,6 +34,22 @@ sidecars:
 
 imagePullSecrets: []
 
+proxy: {}
+#  http_proxy:
+#  no_proxy:
+
+resources:
```

Comment:
think this showed up becasue you had to rebase ,sorry about that, could you remove (the resources got moved to controller.resources so this field isn't used anymore)

Reply:
@wongma7 I just pushed an update.

Reply:
@clarkdoan Hate to ask for it, but would you rebase it once more please? :)